tests/jest/setup.ts

```diff
@@ -28,7 +28,7 @@ jest.doMock('react-native', () => {
                 },
                 AEPEdge: {
                     extensionVersion: jest.fn(() => new Promise(resolve => resolve(''))),
-                    sendEvent: jest.fn(() => new Promise(resolve => resolve(null))),
+                    sendEvent: jest.fn(() => new Promise(resolve => resolve(''))),
```

Comment:
I need to change this to string, then the tests pass.

Reply:
We should check for null before the for loop, and I think that would fix your tests too https://github.com/adobe/aepsdk-react-native/pull/114/files#diff-e010278071c5f1853863f926b699e59cdfbb76d4d01afa67b835f7470b4a5cccR61

Reply:
yes, works now.